mongodb-panache-quickstart/src/test/java/org/acme/mongodb/panache/PersonDTO.java

```diff
@@ -0,0 +1,57 @@
+package org.acme.mongodb.panache;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+
+import java.time.LocalDate;
+
+class PersonDTO {
```

Comment:
Is it really needed ?
We want to keep the quickstart as close to the guide as possible, and we don't use such DTO in the guide.